### DISCUSSION TITLE
Tailwind CSS: t.startsWith is not a function

### DISCUSSION THREAD

**Discussion by zadigg** [tailwindlabs/tailwindcss#5831] [Help]
It was working fine but after i add these three plugins 
plugins: [
    require("@tailwindcss/forms"),
    require("tailwind-scrollbar"),
    require("tailwind-scrollbar-hide"),
  ],

I get this error,  the tailwind works fine but I stop getting the CSS IntelliSense 
![image](https://user-images.githubusercontent.com/62830716/137953575-c3d86980-ff57-4743-a522-43ac7474360d.png)

it's hard to use it without IntelliSense, Any kind of help is appreciated 
Thanks in advance

**Comment by dance2die** [comment]
Can you check if your NodeJS meets the min-version, v12.13?  (You can find out with `node --version` in console)

https://tailwindcss.com/docs/upgrading-to-v2#upgrade-to-node-js-12-13-or-higher

---

Also, do you have a repo you can share for others to reproduce?

**Reply by zadigg** [reply]
My node version is v14.17.6, i was using it an hour ago everything works fine, but i get this error all of a sudden
